### PR TITLE
update makefile - fix win and linux builds on GHA

### PR DESCRIPTION
**Pull request by ph4r05**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,15 +17,17 @@ CLIENT_LDFLAGS=${DEFAULT_LDFLAGS} -X main.defaultHost=${DEFAULT_HOST}
 #BUILD_TIME=`date +%FT%T%z`
 
 # keymaster client requires special tags on linux
-#EXTRA_BUILD_FLAGS 
+EXTRA_BUILD_FLAGS?=
```
Comment (ph4r05) [inline Makefile:20]: this enables to pass additional build flags, required for GHA

**Review by rgooch — APPROVED** [review]

**Comment by ph4r05** [comment]
Thanks for the review @rgooch. I cannot merge, pls merge when you have a minute @cviecco @rgooch. Thanks! :)

**Comment by ph4r05** [comment]
Squash & merge is probably the best :)

**Comment by rgooch** [comment]
@ph4r05: tests failed: https://github.com/Cloud-Foundations/keymaster/actions/runs/8970534255

**Comment by ph4r05** [comment]
@rgooch that is weird! I think I was not touching code that should affect this. Can we rerun failed flows to make sure it is not a transient error?

**Comment by rgooch** [comment]
Yes, that fixed it. @cviecco: FYI.

**Pull request merged**